.vtexignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 .eslintrc
 .gitignore
 README.md
+./**/*.test.ts
```

Comment:
You mentioned you were having problems with this not working to ignore test files in VTEX IO's build. I also had trouble with this the last time I set up a VTEX IO repo with tests, and the way it worked for me was:

```suggestion
**/*.test.*
```

You can see the whole .vtexignore file [here](https://github.com/vtex/buyer-organizations-supplier/pull/40/files#diff-493b51e38bdabe924ce34da582e5e8345ff33831c58186f0cb03e75099b970d7R16) as well in case you have trouble with other types of files too.

Reply:
OK, let me change. I found it strange because locally it is works to link in workspace.